sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/BlobContainerGetPropertiesHeaders.java

```diff
@@ -21,7 +21,7 @@
  */
 @JacksonXmlRootElement(localName = "Container-GetProperties-Headers")
 @Fluent
-public final class ContainerGetPropertiesHeaders {
+public final class BlobContainerGetPropertiesHeaders {
```

Comment:
Codegen class, these changes will be reverted in the next generation.

Reply:
Reverted